sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.0.2 (Unreleased)
+
+- Share a single httpClient among clients. [PR #6657](https://github.com/Azure/azure-sdk-for-js/pull/6657)
```

Comment:
I like it, though the following might be slightly more concise/clear? Thoughts...

- Service clients now share a single http client by default. 
  - Previously, a new http client was created for each service client if none was provided. This could result in TCP port exhaustion under heavy usage with the `keepAlive` option enabled because each http client has its own persistent TCP connection. This change creates a single http client instance which shared among all service clients by default.

Reply:
would "Service clients" cause confusion with BlobServiceClient?

Reply:
Possible, but I think adding `service` to `client` helps disambiguate this usage of client vs an http client. Feel free to take or leave the suggestions though!